mesa/space.py

```diff
@@ -526,6 +535,9 @@ class MultiGrid(_Grid):
     bottom-left and [width-1, height-1] is the top-right. If a grid is
     toroidal, the top and bottom, and left and right, edges wrap to each other.
 
+    Unlike SingleGrid, MultiGrid does not inherently track empty cells due to
```

Comment:
`empties` are tracked in `MultiGrid`. `MultiGrid` inherits from `_Grid`, which defines the `empties` property.
```
In [1]: import mesa
g =
In [2]: g = mesa.space.MultiGrid(5,5,False)

In [3]: g.empties
Out[3]:
{(0, 0),
 (0, 1),
...
```

Reply:
You're right, will update that

Reply:
Updated!